jplag.endToEndTesting/src/test/java/de/jplag/end_to_end_testing/JavaEndToEndTest.java

```diff
@@ -50,48 +59,82 @@ public void teardown() throws IOException {
      * @return the classes to be tested with the corresponding identifier
      */
     private static Stream<Arguments> normalizationLevelTestArguments() {
-        return Stream.of(Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo1.java"}, 0),
-                Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo2.java"}, 1),
-                Arguments.of((Object) new String[] {"SortAlgo1.java", "SortAlgo2.java"}, 2));
+        return Stream.of(Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo1.java"}),
+                Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo2.java"}),
+                Arguments.of((Object) new String[] {"SortAlgo1.java", "SortAlgo2.java"}));
     }
 
     /**
      * Test cases for the token generation level
      * @return the classes to be tested with the corresponding identifier
      */
     private static Stream<Arguments> tokenGenerationLevelTestArguments() {
-        return Stream.of(Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo3.java"}, 0),
-                Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo4.java"}, 1),
-                Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo4d1.java"}, 2));
+        return Stream.of(Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo3.java"}),
+                Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo4.java"}),
+                Arguments.of((Object) new String[] {"SortAlgo.java", "SortAlgo4d1.java"}));
     }
 
     /**
      * Inserting comments or empty lines (normalization level) -> id = 0 Changing variable names or function names
      * (normalization level) -> id = 1 Inserting comments or empty lines (normalization level) -> id = 2
      * @param testClassNames Plagiarized classes names in the resource directorie which are needed for the test
-     * @param testIdentifier name of the testId to load and identify the stored results
      * @throws IOException is thrown in case of problems with copying the plagiarism classes
      * @throws ExitException in case the plagiarism detection with JPlag is preemptively terminated would be of the test.
+     * @throws NoSuchAlgorithmException when no hash algorithm could be found
+     * @throws NameNotFoundException if the no filenames cloud be found in the JPlagCOmparison object
      */
```

Comment:
Good that you explicitly specify what kind of plagiarism actually happens in the files. But in the method header it is a bit hard to discover from my opinion. Maybe we add the kind of performed plagiarism as a comment at the start of each source file? If the original submission also has a comment that we could even say that it doesn't change the structure, but comments should not be tokenized anyways.

Reply:
was removed